networks/ai85-autoencoder.yaml

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
Good to have a comment at the beginning to describe the model, i.e. model type, input size, dataset etc.

Reply:
Updated